charts/telepresence/templates/deployment.yaml

```diff
@@ -41,14 +41,23 @@ spec:
             value: "443"
           - name: CLUSTER_ID
             value: {{ .Values.clusterID }}
+          {{- if (and .Values.agentInjector.create .Values.agentInjector.agentImage.registry) }}
```

Comment:
kind of a nit: since we have a default for `agentInjector.agentImage.registry`, I'd rather just always use that value here instead of doing an if/else and using `image.registry` in the `else` case... does that seem fair?